### PR TITLE
docs(pdk) regenerate PDK documentations to include the new

### PR DESCRIPTION
**Pull request by dndx**
`kong.log.serialize` function

### CODE REVIEW

File: app/2.1.x/pdk/kong.log.md [hunk 1]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+ * `request.querystring` - request query strings
+ * `request.size` - size of request
+ * `request.url` and `request.uri` - URL and URI of request
+ * `response.headers` - response headers
+ * `response.size` - size of response
+ * `response.status` response HTTP status code
+ * `route` - route object matched
+ * `service` - service object used
+ * `started_at` - timestamp this request came in, in milliseconds
+ * `tries` - upstream information, this is an array and if balancer retries occurred, will contain more than one entry
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:289]:
```suggestion
 * `tries` - Upstream information; this is an array and if any balancer retries occurred, will contain more than one entry
```

File: app/2.1.x/pdk/kong.log.md [hunk 2]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+ * `upstream_uri` - request URI sent to upstream
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:290]:
```suggestion
 * `upstream_uri` - request URI sent to Upstream
```

File: app/2.1.x/pdk/kong.log.md [hunk 3]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+
+ The following fields are only present in an authenticated request (with consumer):
+
+ * `authenticated_entity` - credential used for authentication
+ * `consumer` - consumer entity accessing the resource
+
+ The following fields are only present in a TLS/HTTPS request
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:297]:
```suggestion
 The following fields are only present in a TLS/HTTPS request:
```

File: app/2.1.x/pdk/kong.log.md [hunk 4]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+ * `request.tls.version` - TLS/SSL version used by the connection
+ * `request.tls.cipher` - TLS/SSL cipher used by the connection
+ * `request.tls.client_verify` - mTLS validation result. Contents are the same as described in [$ssl_client_verify](https://nginx.org/en/docs/http/ngx_http_ssl_module.html#var_ssl_client_verify)
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:300]:
```suggestion
 * `request.tls.client_verify` - mTLS validation result. Contents are the same as described in [$ssl_client_verify](https://nginx.org/en/docs/http/ngx_http_ssl_module.html#var_ssl_client_verify).
```

File: app/2.1.x/pdk/kong.log.md [hunk 5]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+
+ Warning: this function may return sensitive data (e.g. API keys).
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:302]:
```suggestion
**Warning:** This function may return sensitive data (e.g., API keys).
```

File: app/2.1.x/pdk/kong.log.md [hunk 6]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+ Consider filtering before writing it to unsecured locations.
+
+ To see what content are there on your setup, enable any of the logging
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:305]:
```suggestion
 To see what content is present in your setup, enable any of the logging
```

File: app/2.1.x/pdk/kong.log.md [hunk 7]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+ plugins (e.g. `file-log`) and the output written to log file is the table
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:306]:
```suggestion
 plugins (e.g. ,`file-log`) and the output written to the log file is the table
```

File: app/2.1.x/pdk/kong.log.md [hunk 8]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+ returned by this function JSON encode
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:307]:
```suggestion
 returned by this function JSON-encoded.
```

File: app/2.1.x/pdk/kong.log.md [hunk 9]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+
+
 **Phases**
 
 * log
 
+**Returns**
+
+* `table` table containing request information
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:316]:
```suggestion
* `table` - table that contains request information.
```

File: app/2.1.x/pdk/kong.log.md [hunk 10]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+ * `client_ip` - client IP address in textual format
+ * `latencies` - request/proxy latencies
+ * `request.headers` - request headers
+ * `request.method` - request method
+ * `request.querystring` - request query strings
+ * `request.size` - size of request
+ * `request.url` and `request.uri` - URL and URI of request
+ * `response.headers` - response headers
+ * `response.size` - size of response
+ * `response.status` response HTTP status code
```
Comment (tlblessing) [inline app/2.1.x/pdk/kong.log.md:285]:
```suggestion
 * `response.status` - response HTTP status code
```

File: app/2.1.x/pdk/kong.log.md (hunk 4 above)
Comment (tlblessing) [reply app/2.1.x/pdk/kong.log.md:300]: Should be consistent. Probably add periods to all entries.

**Comment by lena-larionova** [comment]
> when does the Enterprise side get this update? https://docs.konghq.com/enterprise/1.5.x/pdk/kong.log/

Technically we should be in sync now, I hope? The problem is the enterprise PDK doc isn't autogenerated like the core doc (yet). So if these changes apply to Enterprise, we need to manually update the relevant doc on that side.

**Comment by dndx** [comment]
@lena-larionova @tlblessing yes this change is coming to EE 2.1 as well.

**Comment by dndx** [comment]
Hey @tlblessing,

Thanks for the reviews! I addressed everything except for the `@treturn table` one. The format was actually generated by the document generator, but I did changed the wording a bit so hopefully it is less confusing to read now.

See https://deploy-preview-2167--kongdocs.netlify.app/2.1.x/pdk/kong.node/#kongnodeget_memory_statsunit-scale for another example of the generated return values.

We could add dashes to the return values, but it is probably better be done on the document generator in a separate PR because that updates everywhere at once.

Let me know what you think :)

File: app/2.1.x/pdk/kong.log.md [hunk 11]
```diff
@@ -270,14 +270,56 @@ kong.log.inspect.off()
...
+ The following fields are only present in a TLS/HTTPS request:
+ * `request.tls.version` - TLS/SSL version used by the connection.
+ * `request.tls.cipher` - TLS/SSL cipher used by the connection.
+ * `request.tls.client_verify` - mTLS validation result. Contents are the same as described in [$ssl_client_verify](https://nginx.org/en/docs/http/ngx_http_ssl_module.html#var_ssl_client_verify).
+
+ **Warning:** This function may return sensitive data (e.g., API keys).
+ Consider filtering before writing it to unsecured locations.
+
+ To see what content is present in your setup, enable any of the logging
+ plugins (e.g. ,`file-log`) and the output written to the log file is the table
```
Comment (gszr) [inline app/2.1.x/pdk/kong.log.md:306]:
```suggestion
 plugins (e.g., `file-log`) and the output written to the log file is the table
```

**Pull request merged**